File: clients/rust/src/anonymous_schema19.rs
// AnonymousSchema19 represents a AnonymousSchema19 model.
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct AnonymousSchema19 {
    #[serde(rename="request")]
    request: serde_json::Value,
    #[serde(rename="unit")]
    unit: Box<crate::SystemdUnit>,
}

impl AnonymousSchema19 {
    pub fn new(request: serde_json::Value, unit: crate::SystemdUnit) -> AnonymousSchema19 {
        AnonymousSchema19 {
        request,
        Box::new(unit),
        }
    }
}

File: clients/typescript/src/AnonymousSchema_19.ts
import {SystemdUnit} from './SystemdUnit';
export class AnonymousSchema_19 {
  private _request: any;
  private _unit: SystemdUnit;

  constructor(input: {
    request: any,
    unit: SystemdUnit,
  }) {
    this._request = input.request;
    this._unit = input.unit;
  }

  get request(): any { return this._request; }
  set request(request: any) { this._request = request; }

  get unit(): SystemdUnit { return this._unit; }
  set unit(unit: SystemdUnit) { this._unit = unit; }
}
